cdbot/cogs/admin.py

```diff
@@ -89,6 +91,50 @@ async def on_member_join(self, member: Member):
             # assign placeholder nickname
             await member.edit(nick=PLACEHOLDER_NICKNAME)
 
+    @command()
+    @has_role(ROOT_ROLE_ID)
```

Comment:
But there are only two roots as of now 🤔

Reply:
So allow Sudos as well?

Reply:
Idk, you'd probably have to ask staff since that'd be their view, but I'm just saying from an availability standpoint :+1: :joy: